### PR TITLE
Add UploadMediaBundle.java

### PR DESCRIPTION
**Pull request by wayneluzw**
Based on [PHP library example](https://github.com/googleads/google-ads-php/blob/e1ff0e4aa64d6c640a76300e8a000dd2486da180/examples/Misc/UploadMediaBundle.php).

### CODE REVIEW

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId) throws IOException {
+
+    byte[] bundleData = ByteStreams.toByteArray(new URL(BUNDLE_URL).openStream());
+
+    // Creates a media bundle file
```
Comment (fiboknacky) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java:95]: As a convention, please add a period at the end of all comments.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
+//
+// Copyright 2019 Google LLC
```
Comment (fiboknacky) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java:2]: 2020 now.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java (hunk 1 above)
Comment (wayneluzw) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java]: Sure, no problem.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        googleAdsClient.getLatestVersion().createMediaFileServiceClient()) {
+      MutateMediaFilesResponse response =
+          mediaFileServiceClient.mutateMediaFiles(Long.toString(customerId), Arrays.asList(op));
+      System.out.printf(
+          "The media bundle with resource name '%s' was added.%n",
+          response.getResults(0).getResourceName());
+    }
+  }
+
+  private static class UploadMediaBundleParams extends CodeSampleParams {
```
[the remark this answers is not in the document]
Comment (wayneluzw) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java]: Done

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Arrays;
+
+/**
+ * This example uploads an HTML5 zip file as a Media Bundle. More information about Media Bundle can
+ * be found at https://developers.google.com/google-ads/api/docs/assets/overview
+ */
+public class UploadMediaBundle {
```
[the remark this answers is not in the document]
Comment (wayneluzw) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java:42]: Done, but there's no changes. Actually I've set this enabled by default in IntelliJ.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+import com.google.protobuf.ByteString;
+import com.google.protobuf.BytesValue;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Arrays;
+
+/**
+ * This example uploads an HTML5 zip file as a Media Bundle. More information about Media Bundle can
```
Comment (nwbirnie) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java:39]: Can abbreviate the start of this sentence to "Uploads an HTML5..."

Comment (wayneluzw) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java]: NP

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+import com.google.protobuf.ByteString;
+import com.google.protobuf.BytesValue;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Arrays;
+
+/**
+ * Uploads an HTML5 zip file as a Media Bundle. More information about Media Bundle can be found at
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java:39]: nit: I'd avoid capitalizing "Media Bundle", and also say in the 2nd sentence:

_More information about media bundles can be found at..._

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+            .setMediaBundle(bundle)
+            .build();
+
+    // Creates a MediaFile operation.
+    MediaFileOperation op = MediaFileOperation.newBuilder().setCreate(file).build();
+
+    // Creates the MediaFile service client.
+    try (MediaFileServiceClient mediaFileServiceClient =
+        googleAdsClient.getLatestVersion().createMediaFileServiceClient()) {
+      MutateMediaFilesResponse response =
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java:119]: Add a line comment explaining this sends a mutate operation to add the bundle, or just change the comment on line 116 to match the [PHP example](https://github.com/googleads/google-ads-php/blob/e1ff0e4aa64d6c640a76300e8a000dd2486da180/examples/Misc/UploadMediaBundle.php#L114).

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId) throws IOException {
+
+    byte[] bundleData = ByteStreams.toByteArray(new URL(BUNDLE_URL).openStream());
```
Comment (jradcliff) [inline google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java:99]: Add line comment explaining this reads the sample media bundle from the URL into a byte array.

Comment (wayneluzw) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java:100]: Sounds good

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java (hunk 7 above)
Comment (wayneluzw) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java:121]: Sounds good.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java (hunk 6 above)
Comment (wayneluzw) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/misc/UploadMediaBundle.java]: Sounds good

**Pull request merged**